### PR TITLE
Stats for organization

### PR DESCRIPTION
**Pull request by jruo**

### CODE REVIEW

File: app/views/stats/index.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h1>Statistics</h1>
+<h1>Statistics<%=" for #{@organization.name}" unless @organization.nil?%> </h1>
```
Comment (jamox) [inline app/views/stats/index.html.erb]: Is this a hack? -- when we would have this page w/o org?

**Comment by jamox** [comment]
LGTM if tests pass + comment above

File: app/views/stats/index.html.erb (hunk 1 above)
Comment (jsopakar) [inline app/views/stats/index.html.erb]: Old functionality to show all courses (for all organizations) stats in /stats was still retained. This was also kept without permission checks, based on irc discussions.

Do we want to get rid of that? Even for admin purposes? Can change if needed.

Comment (mpartel) [inline app/views/stats/index.html.erb]: I think it's fine

**Comment by jamox** [comment]
LGMT

**Comment by jamox** [comment]
Cherry-picked yesterday and now in prod

**Pull request closed without merging**